CSS/CSS Presentation/Problem_4/index.html

```diff
@@ -0,0 +1,239 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Calculator</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <table>
+        <tr class="main-container">
+            <td colspan="7">🖩 Calculator</td>
+            <td><button class="btn-m">🗕</button></td>
+            <td><button class="btn-r">☐</button></td>
+            <td><button class="btn-c">✕</button></td>
+        </tr>
+        <tr class="menu">
+            <td colspan="10"> 
+                <span class="menu-item">View</span>
+                <span class="menu-item">Edit</span>
+                <span class="menu-item">Help</span>
+            </td>
+        </tr>
+        <tr class="result">
+            <td colspan="10">
+                <span>8854455</span>
+            </td>
+        </tr>
+        <tr class="b-num-row">
+            <td colspan="10">
+                <span>0000</span>
+                <span>0000</span>
+                <span>0000</span>
+                <span>0000</span>
+                <span>0000</span>
+                <span>0000</span>
+                <span>0000</span>
+                <span>0000</span>
+            </td>
+        </tr>
+        <tr class="d-num-row">
+            <td colspan="3" class="first">63</td>
+            <td colspan="5" class="second">47</td>
+            <td colspan="3" class="third">32</td>
+        </tr>
+        <tr class="b-num-row">
+            <td colspan="10">
+                <span>0000</span>
+                <span>0000</span>
+                <span>1000</span>
+                <span>0111</span>
+                <span>0001</span>
+                <span>1011</span>
+                <span>1011</span>
+                <span>0111</span>
+            </td>
+        </tr>
+        <tr class="d-num-row">
+            <td colspan="3" class="first">31</td>
+            <td colspan="5" class="second">15</td>
+            <td colspan="3" class="third">0</td>
+        </tr>
+        <tr>
+            <td rowspan="3" colspan="2" class="radio-opt">
+                <input type="radio" id="hex">
```

Comment:
All radio buttons can be ticked with this html. Please make sure that only one radio per group can be ticked.